pymodelextractor/learners/observation_table_learners/generic_lstar_learner.py

```diff
@@ -0,0 +1,239 @@
+from ast import Tuple
+from pythautomata.base_types.sequence import Sequence
+from pymodelextractor.learners.observation_table_learners.generic_observation_table import GenericObservationTable
+from pymodelextractor.learners.learning_result import LearningResult
+from pymodelextractor.teachers.generic_teacher import GenericTeacher
+import time
+import multiprocessing
+from pymodelextractor.utils.time_bound_utilities import timeout
+
+lamda = Sequence()
+
+no_log = 'none'
+info_log = 'info'
+debug_log = 'debug'
+trace_log = 'trace'
+
+class GenericLStarLearner:
+    def __init__(self, model_translator, max_states = None, max_query_lenght = None, max_time = None):
```

Comment:
`max_query_length = -1`
`max_states`=-1
`max_time=-1`
(let's not mix numbers and pointers)